reference/shaders-hlsl/comp/global-invocation-id-writable-ssbo-in-function.comp

```diff
@@ -0,0 +1,36 @@
+struct myBlock
+{
+    int a;
+    float b[1];
```

Comment:
This array is just because I am using the same source shader that is in [`shaders-msl/comp/global-invocation-id-writable-ssbo-in-function.comp`](https://github.com/KhronosGroup/SPIRV-Cross/blob/master/shaders-msl/comp/global-invocation-id-writable-ssbo-in-function.comp)

The implicit unsized array is seen in changing `myStorage` to `myStorage[0]` [here](https://github.com/KhronosGroup/SPIRV-Cross/pull/220/files#diff-a268d1a6fec87b22037be5bf40f35bc8R27)